### PR TITLE
Issue #4613: Cleanup layout.context.admin

### LINKED ISSUE #4613: Clean up layout.context.admin
**Edit:**
Consensus seems to be to keep the context code in separate files after all. Issue title updated. PR attached to make the code  a bit more legitimate. Added `hook_menu()` `file` entries and removed the temporary `module_load_include()` calls

Original post below:
Just realised a small oops: during developing custom contexts, I put all the new code into a "temporary" `layout.contexts.admin.inc` file and included it with `module_load_include('inc', 'layout', 'layout.context.admin')` (and also did the opposite in the other file too, more oops) because `layout.admin.inc` is huge and hard to work with. That seems to have been merged as is. It is administrative code though, so has all right to be in `layout.admin.inc`. Should we merge it in there or leave it out as a special case?

__
PR by @docwilmot: https://github.com/backdrop/backdrop/pull/3310

### PR DESCRIPTION
**Pull request by docwilmot**
Fixes https://github.com/backdrop/backdrop-issues/issues/4613

### CODE REVIEW

File: core/modules/layout/layout.context.admin.inc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?php
-module_load_include('inc', 'layout', 'layout.admin');
-
+/**
```
Comment (indigoxela) [inline core/modules/layout/layout.context.admin.inc:2]: Thanks for adding the doc comment, but there should be one blank line before it. (Sorry for being so picky. :wink: )

Comment (herbdool) [reply core/modules/layout/layout.context.admin.inc:2]: Actually I think it's not standard in this project not to have a space. So this is fine. See examples like admin_bar.module.

Comment (indigoxela) [reply core/modules/layout/layout.context.admin.inc:2]: Ah, OK, then ignore it. :wink: It's just that phpcs nags because of it.

Comment (herbdool) [reply core/modules/layout/layout.context.admin.inc:2]: I'm not sure why there's no space, but tis the custom. Oh well.

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.17.x.

**Pull request merged**